addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14599,10 +14599,10 @@ msgctxt "#36140"
 msgid "Choose which temperature unit is used for displaying temperatures in the user interface."
 msgstr ""
 
-#. Description of setting "Videos -> Library -> Show plot for unwatched items" with label #20369
+#. Description of setting "Videos -> Library -> Show plot for unwatched items (Hides plot and thumb when disabled)" with label #20369
 #: system/settings/settings.xml
 msgctxt "#36141"
-msgid "Show plot information for unwatched media in the video library."
+msgid "[Enabled] Show plot information for unwatched media in the video library.[CR][Disabled] Hides plot and thumbnail to prevent spoilers."
```

Comment:
Actually no this is for movies and episodes.

So movies only plot is hidden/shown Episodes plot/thumb is hidden or shown.